### DISCUSSION TITLE
Calling a reusable request prior running tests

### DISCUSSION THREAD

**Discussion by axeloz** [stepci/stepci#201] [General]
Hi there, 
Thanks for StepCI. Very nice and useful 👍🏻
I want to parallelize the tests and splitting the tests in many subtests. My API requires to be logged in with a bearer token. 
Is there a way to set an order or some kind of dependancies (ie do not run that tests unless this test has completed). 

To be clear, my login request must also run first. Then the others. I could easily copy/paste the login request into each test but is there a better way to do this?
Thanks

**Comment by mishushakov** [comment]
Yes, see this: https://docs.stepci.com/guides/organising-workflows.html
